projects/plugins/jetpack/modules/waf.php

```diff
@@ -11,3 +11,58 @@
  *
  * @package automattic/jetpack
  */
+
+use Automattic\Jetpack\Waf\Waf_Runner;
+use Automattic\Jetpack\Waf\Waf_Standalone_Bootstrap;
+
+/**
+ * Jetpack waf module class.
+ */
+class Jetpack_Waf_Module {
+
+	/**
+	 * Instance of the class.
+	 *
+	 * @var Jetpack_Waf_Module()
+	 */
+	private static $instance = null;
+
+	/**
+	 * Singleton implementation
+	 *
+	 * @return object
+	 */
+	public static function instance() {
```

Comment:
So seeing how we are using the Singleton pattern here I wanted to ask about it. Do you think we can instead rely on the Config class to make sure we're doing this only once?

There's a good example of how we're doing it in Publicize that was added recently:
https://github.com/Automattic/jetpack/pull/23786/files

That would probably look better and take less code than a Singleton. Please let me know if you need help with this.